src/main/java/tc/oc/pgm/modules/SoundsMatchModule.java

```diff
@@ -55,7 +55,9 @@ private void playRaindrop(Competitor competitor) {
   public void onMatchPlayerDeath(MatchPlayerDeathEvent event) {
     ParticipantState killer = event.getKiller();
     MatchPlayer victim = event.getVictim();
-    if (killer != null && !killer.getId().equals(victim.getId())) {
+    if (killer != null
+        && !killer.getId().equals(victim.getId())
```

Comment:
remove the own-kill check if we have a same-party check

Reply:
Yeah thought about it